rct229/rulesets/ashrae9012019/section19/section19rule3.py

```diff
@@ -12,49 +12,86 @@ class Section19Rule3(PartialRuleDefinition):
-            is_primary_rule=False,
+            is_primary_rule=True,
             required_fields={
                 "$.ruleset_model_descriptions[*]": ["weather"],
             },
         )
 
     def get_calc_vals(self, context, data=None):
         rpd_b = context.BASELINE_0
+        rpd_p = context.PROPOSED
+
         weather_b = rpd_b["ruleset_model_descriptions"][0]["weather"]
+        weather_p = rpd_p["ruleset_model_descriptions"][0]["weather"]
+
         return {
-            "cooling_design_day_type": weather_b.get("cooling_design_day_type"),
-            "heating_design_day_type": weather_b.get("heating_design_day_type"),
+            "cooling_design_day_type_b": weather_b.get(
+                "cooling_dry_bulb_design_day_type"
+            ),
+            "heating_design_day_type_b": weather_b.get(
+                "heating_dry_bulb_design_day_type"
+            ),
+            "evaporation_wet_bulb_design_day_type_b": weather_b.get(
+                "evaporation_wet_bulb_design_day_type"
+            ),
+            "cooling_design_day_type_p": weather_p.get(
+                "cooling_dry_bulb_design_day_type"
+            ),
+            "heating_design_day_type_p": weather_p.get(
+                "heating_dry_bulb_design_day_type"
+            ),
+            "evaporation_wet_bulb_design_day_type_p": weather_p.get(
+                "evaporation_wet_bulb_design_day_type"
+            ),
         }
 
-    def applicability_check(self, context, calc_vals, data):
-        cooling_design_day_type = calc_vals["cooling_design_day_type"]
-        heating_design_day_type = calc_vals["heating_design_day_type"]
+    def rule_check(self, context, calc_vals=None, data={}):
+        cooling_design_day_type_b = calc_vals["cooling_design_day_type_b"]
+        heating_design_day_type_b = calc_vals["heating_design_day_type_b"]
+        evaporation_wet_bulb_design_day_type_b = calc_vals[
+            "evaporation_wet_bulb_design_day_type_b"
+        ]
+        cooling_design_day_type_p = calc_vals["cooling_design_day_type_p"]
+        heating_design_day_type_p = calc_vals["heating_design_day_type_p"]
+        evaporation_wet_bulb_design_day_type_p = calc_vals[
+            "evaporation_wet_bulb_design_day_type_p"
+        ]
 
-        return (cooling_design_day_type is None or heating_design_day_type is None) or (
-            cooling_design_day_type == COOLING_DESIGN_DAY.COOLING_1_0
-            and heating_design_day_type == HEATING_DESIGN_DAY.HEATING_99_6
+        return (
+            cooling_design_day_type_b
+            == evaporation_wet_bulb_design_day_type_b
+            == COOLING_DESIGN_DAY.COOLING_1_0
+            and heating_design_day_type_b == HEATING_DESIGN_DAY.HEATING_99_6
+        ) and (
+            cooling_design_day_type_p
+            == evaporation_wet_bulb_design_day_type_p
+            == COOLING_DESIGN_DAY.COOLING_1_0
+            and heating_design_day_type_p == HEATING_DESIGN_DAY.HEATING_99_6
```

Comment:
As discussed in the meeting, the understanding of the RDS is as following:

if baseline pass & proposed pass 
  → pass
else if proposed failed
  → print out failed message
else 
  → fail

Reply:
FYI, I haven't updated the RDS yet because it hasn't been merged to develop. Once it's merged, I'll update it.